packages/tangle/tangle.go

```diff
@@ -125,6 +117,13 @@ func (t *Tangle) Setup() {
 	t.ConsensusManager.Setup()
 	t.TipManager.Setup()
 
+	// increase scheduler rate if the node is not synced at start
+	if !t.Synced() {
+		rate := t.Scheduler.Rate()
+		rate -= rate / 2 // 50% increase
+		t.Scheduler.SetRate(rate)
+	}
```

Comment:
Considering that we bypass the scheduler for all the messages older than 5 minutes, do we still need this at all?

Reply:
Let's remove it!